src/Configuration.c

```diff
@@ -21,8 +81,43 @@ static const char* set_server_url(cmd_parms* cmd, void* cfg, const char* arg)
     return NULL;
 }
 
+static const char* set_use_jku(cmd_parms* cmd, void* cfg, const char* arg)
+{
+    configuration.use_jku = arg && wal_stricmp(arg,"true")==0;
```

Comment:
Could you please run "clang-format".

Reply:
sure.
Btw: I will provide an update with additional features RS512 support and so that it also works on RHEL 7.9. Yet this will need some time (most of it is done, final testing is open and I'm away the next two weeks....).